src/Datagrid/Datagrid.php

```diff
@@ -418,12 +420,22 @@ public function getQueryBuilder()
     /**
      * Store values for filters so user can change page and keep his filters
      * Values are stored per admin
+     * @TODO support entities that don't have getId() as method to retrieve the primary id
```

Comment:
Couldn't we define some kind of "primary key" argument (which would default to 'id') then use accessor instead of directly calling getId() ?